README.md

```diff
@@ -1,6 +1,197 @@
 # Portals
-Portals is a very early proposal for enabling seamless navigations between sites or pages. In particular, this proposal enables a page to show another page as an inset and perform a seamless transition between an inset state and a navigated state.
 
-See the **[explainer](https://github.com/WICG/portals/blob/master/explainer.md)**.
+Portals are a proposed new HTML element which enables seamless navigations between pages. In particular, the `<portal>` element enables a page to show another page as an inset, and then "activate" it to perform a seamless transition to a new state, where the formerly-inset page becomes the top-level document.
 
-See the **[early draft spec](https://wicg.github.io/portals/)**.
+Portals encompass some of the use cases that iframes currently do, but with better security, privacy, and ergonomics properties. And via the activation operation, they enable new use cases like preloading or navigation transitions.
```

Comment:
I wonder if it'd be worth mentioning the "super-links" framing here.

Reply:
Given the concessions we need to make for cross-origin portals, I'm a little nervous about about saying they're better than iframes. They comes with additional restrictions compared to iframes, so they can be activated without compromising privacy. But ergonomics wise, things like the rendering style will make them less-useful for some of the things folks use iframes for today.

Reply:
Right, I tried to be explicit it only covers some of the use cases iframes do, with some more details below.